demos/embed/player.html

```diff
@@ -60,9 +60,19 @@
         import { attrs } from "../assets/options.js";
 
         const localAttrs = {...attrs, ...{
+            // OM SDK Sample Pre-roll
+            // adtagurl: "https://pubads.g.doubleclick.net/gampad/ads?iu=/21775744923/external/omid_ad_samples&env=vp&gdfp_req=1&output=vast&sz=640x480&description_url=http%3A%2F%2Ftest_site.com%2Fhomepage&vpmute=0&vpa=0&vad_format=linear&url=http%3A%2F%2Ftest_site.com&vpos=preroll&unviewed_position_start=1&correlator=",
             // Can set any custom attributes here
             // **THEMES
             theme: "modern",
+            ads_request_options: {
+                // adsResponse: "SOME XML",
+                omidAccessModeRules: {
+                    'OmidVerificationVendor.GOOGLE': 'OmidAccessMode.FULL', // 9: 'full',
+                    'OmidVerificationVendor.OTHER': 'OmidAccessMode.LIMITED', // 1: 'limited',
+                    'OmidVerificationVendor.MOAT': 'OmidAccessMode.DOMAIN',
```

Comment:
Can we also give IAS and DV full access as well?

Reply:
hey @jwellborn-kargo it's as example used here. It's not affect on player itself. 

In the main player these will be set by default or after merging you'll be able add settings as 

```
ads_request_options: {
  omidAccessModeRules: {
    'OmidVerificationVendor.INTEGRAL_AD_SCIENCE': 'OmidAccessMode.FULL',
    'OmidVerificationVendor.DOUBLEVERIFY': 'OmidAccessMode.FULL',
 }
}
```

all available options for today are listed here: https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/reference/namespace/google.ima#google.ima.OmidVerificationVendor

Reply:
Thanks for clarifying.